Samples/ResourceManagement/cpp-winui/winui_desktop_packaged_app_cpp/packages.config

```diff
@@ -2,6 +2,6 @@
 <packages>
   <package id="Microsoft.VCRTForwarders.140" version="1.0.7" targetFramework="native" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.210714.1" targetFramework="native" />
-  <package id="Microsoft.Windows.SDK.BuildTools" version="10.0.20348.19" targetFramework="native" />
-  <package id="Microsoft.WindowsAppSDK" version="1.0.0-preview1" targetFramework="native" />
+  <package id="Microsoft.Windows.SDK.BuildTools" version="10.0.22499-preview" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK" version="1.0.0-20211111.1-CI-stable" targetFramework="native" />

```

Comment:
I assume we will wait for the 1.0.0 release and update these versions to "1.0.0"? This version is internal only

Reply:
Good point Angela :)